kubelet-to-gcm/README.md

```diff
@@ -0,0 +1,85 @@
+# addon-resizer
```

Comment:
I think this is still not fixed. This file describes addon resizer, not kubelet-to-gcm.